### ISSUE TITLE
Allow for setting count/aggregation on the facet filter component

### ISSUE THREAD

**Issue by esjewett** [humanitiesplusdesign/palladio#52]

**Comment by cncoleman** [comment]
Hmmm. I'm not quite sure what this means. I was going to create an enhancement issue for the facet filter to show ([number selected out of] / [total number]) for each dimension.

We need to let people know at a glance how many of a thing they are seeing in a view. We can do this easily with FF instead of mucking up the interface with more information.

**Comment by esjewett** [comment]
Yes, that's good. This is a different issue - that when you are using the facet filter as a component there is no way to select which aggregation (count vs. sum) to use.

**Comment by cncoleman** [comment]
i see.
